packages/yoastseo/spec/scoring/assessments/seo/ProductSKUAssessmentSpec.js

```diff
@@ -198,69 +176,83 @@ describe( "a test for the applicability of the assessment", function() {
 		const assessment = new ProductSKUAssessment( { assessVariants: true } );
 		const customData = {
 			canRetrieveGlobalSku: false,
 			hasGlobalSKU: false,
 			hasVariants: false,
-			productType: "simple",
+			productType: "external",
 		};
 		const paperWithCustomData = new Paper( "", { customData } );
 		const isApplicable = assessment.isApplicable( paperWithCustomData );
 
 		expect( isApplicable ).toBe( false );
 	} );
 
-	it( "is not applicable when the SKU of an external product cannot be detected", function() {
+	it( "is applicable when we want to assess variants and when the global SKU cannot be detected on product with variants," +
+		"but the variant SKU can", function() {
 		const assessment = new ProductSKUAssessment( { assessVariants: true } );
 		const customData = {
+			canRetrieveVariantSkus: true,
 			canRetrieveGlobalSku: false,
 			hasGlobalSKU: false,
-			hasVariants: false,
-			productType: "external",
+			hasVariants: true,
+			productType: "variable",
 		};
 		const paperWithCustomData = new Paper( "", { customData } );
 		const isApplicable = assessment.isApplicable( paperWithCustomData );
 
-		expect( isApplicable ).toBe( false );
+		expect( isApplicable ).toBe( true );
 	} );
 
-	it( "is applicable when the SKU cannot be detected on product with variants", function() {
+	it( "is not applicable when we want to assess variants and the SKU of at least one variant cannot be detected on product with variants", function() {
 		const assessment = new ProductSKUAssessment( { assessVariants: true } );
 		const customData = {
-			canRetrieveGlobalSku: false,
+			canRetrieveVariantSkus: false,
 			hasGlobalSKU: false,
 			hasVariants: true,
 			productType: "variable",
 		};
 		const paperWithCustomData = new Paper( "", { customData } );
 		const isApplicable = assessment.isApplicable( paperWithCustomData );
 
+		expect( isApplicable ).toBe( false );
+	} );
+
+	it( "is applicable when we don't want to assess variants and the product doesn't have variants", function() {
+		const assessment = new ProductSKUAssessment( { assessVariants: false } );
+		const customData = {
+			hasGlobalSKU: false,
+			hasVariants: false,
+			productType: "simple",
+		};
+		const paperWithCustomData = new Paper( "", { customData } );
+		const isApplicable = assessment.isApplicable( paperWithCustomData );
+
 		expect( isApplicable ).toBe( true );
 	} );
 
-	it( "is not applicable when the SKU of at least one variant cannot be detected on product with variants", function() {
-		const assessment = new ProductSKUAssessment( { assessVariants: true } );
+	it( "is not applicable when we don't want to assess variants and the product has variants", function() {
+		const assessment = new ProductSKUAssessment( { assessVariants: false } );
 		const customData = {
-			canRetrieveVariantSkus: false,
 			hasGlobalSKU: false,
 			hasVariants: true,
-			productType: "variable",
+			productType: "simple",
```

Comment:
```suggestion
			productType: "variable",
```
This is a weird situation. It should not be possible for a simple product to be simple and have-variants (unless this is shopify-specific and I have no knowledge about it).
The product-type should not influence the applicability. If the productType is `variable` and applicability would be `true`, then there is a different problem. So I would suggest to change this back.

Reply:
So product type is not actually a thing in Shopify, like it is in Woo. But because the scoring criteria check product type, we decided to still add the productType property to Shopify data, and set it to "simple". It doesn't change so all Shopify product types are "simple".

I realize this is a slightly confusing solution, but it was the simplest in the sense that it didn't require any refactoring or adding additional code. Let me know if you have a suggestion for a different approach though :)